### DISCUSSION TITLE
Copy URL Path to Folder in File Cabinet

### DISCUSSION THREAD

**Discussion by dd-manderson** [burkybang/SuiteAdvanced-Feedback#132] [Ideas]
In the file cabinet, it would be helpful if there was a button next to a folder that you could click to create a URL that would open that folder directly if you pasted the copied URL into a new tab. There is no easy way to create a shortcut to a NetSuite file cabinet folder or file.  Anything to help with that would be great.

![file-cabiet-button](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/160878598/01668e5b-052d-459b-a7fa-e98b3786674f)

**Comment by dd-manderson** [comment]
Here's a help doc on creating file cabinet shortcuts 

https://docs.oracle.com/en/cloud/saas/netsuite/ns-online-help/section_N541665.html#subsect_163723979344

**Comment by burkybang** [comment]
When navigating the file cabinet with SuiteAdvanced, the URL in your browser changes to the current folder's URL. You should be able to just copy that.

**Reply by dd-manderson** [reply]
Hmm.. I'm not seeing that behavior

**Reply by dd-manderson** [reply]
OK, nevermind.  I see it now.  I think I had the Drag and Drop file cabinet window open and I was copying and pasting from that.

**Reply by burkybang** [reply]
You should see the "folder" parameter change to the current folder's internal ID, no?

**Reply by burkybang** [reply]
Oh true. Yeah it won't work with the Drag and Drop bundle's file cabinet, but SuiteAdvanced adds drag and drop uploading to the normal file cabinet.
